bundles/org.openhab.binding.knx/src/test/java/org/openhab/binding/knx/internal/i18n/KNXTranslationProviderTest.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.knx.internal.i18n;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.junit.jupiter.api.Test;
+
+import tuwien.auto.calimero.KNXException;
+import tuwien.auto.calimero.link.KNXLinkClosedException;
+
+/**
+ *
+ * @author Holger Friedrich - initial contribution
+ *
+ */
+@NonNullByDefault
+public class KNXTranslationProviderTest {
+    static final String UNKNOWN = "unknown text";
+    static final String UNKNOWN_PATTERN = "unknown text {0}";
+    static final String UNKNOWN_FIVE = "unknown text 5";
+    static final String UNKNOWN_NULL = "unknown text null";
+    static final String KNOWN_DAY_KEY = "dynamic-channel.automation.weekdays";
+    static final String KNOWN_DAY_VALUE = "Weekdays";
+    static final String KNOWN_DAY_PATTERN_KEY = "dynamic-channel.automation.after-sunset";
+    static final String KNOWN_DAY_PATTERN_VALUE = "{0} after sunset";
+    static final String KNOWN_DAY_FIVE_VALUE = "5 after sunset";
+    static final String KNOWN_DAY_ORIG_VALUE = "null after sunset";
+    static final String KNOWN_DAY_NULL_VALUE = "null after sunset";
```

Comment:
I'm wondering if it would make sense with some strings from the knx binding here? The mentioned keys do not exist in as properties, so it seems quite artificial.